dicom_header_editor.py

```diff
@@ -0,0 +1,48 @@
+from pydicom import dcmread
+from os import listdir, system
+from os.path import isfile, join
+import datetime
+
+src_folder = 'src_image'
+dest_folder = 'dest_images'
+base_file_name = 'custom_dicom_'
```

Comment:
Please, use capital letters for constant variables.

Reply:
Made these constants configurable by the user via script arguments: https://github.com/darwinai/covidnet_integration/pull/3/commits/ad885f7bc3e24d4f90f66a30825058ce5c075534